modules/ocf_stats/files/munin/mail-munin-alert

```diff
@@ -78,7 +78,7 @@ def send_alert(host, plugin, mail_to=MAIL_TO):
         ldap_entry=ldap_entry,
         dns_names=dns_names,
     )
-    send_mail(mail_to, subject, body, MAIL_FROM)
+    send_mail(to=mail_to, subject=subject, body=body, cc=None, sender=MAIL_FROM)
```

Comment:
no need to specify `cc=None`

Reply:
I would suggest writing this with positional arguments for required params and keyword arguments for optional params, like:

    send_mail(mail_to, subject, body, sender=MAIL_FROM)

When ocf/ocflib#114 was merged, I slightly feared that this would happen. I should have raised my concerns then.

Reply:
I know but I prefer to list all the arguments in cases like these, makes it slightly easier when reading the code again in the future after a long time.